wolfssl/wolfcrypt/types.h

```diff
@@ -31,278 +31,286 @@ decouple library dependencies with standard string, memory and so on.
 #ifndef WOLF_CRYPT_TYPES_H
 #define WOLF_CRYPT_TYPES_H
 
-    #include <wolfssl/wolfcrypt/settings.h>
-    #include <wolfssl/wolfcrypt/wc_port.h>
-    #include <wolfssl/wolfcrypt/oid_sum.h>
+#include <wolfssl/wolfcrypt/settings.h>
```

Comment:
These changes are mostly whitespace and line length. The new addition is at the bottom for `struct CertSignCtx`.